rust/ranked-prolly-tree/README.md

```diff
@@ -0,0 +1,153 @@
+# ranked-prolly-tree
+
+
+`ranked-prolly-tree` (RPT) is a key-value store implemented as a [prolly tree], utilizing a flexible content-addressed block storage backend. RPT is designed to be the foundation of a lazy database, utilizing partial sync on demand.
+
+## Usage
+
+```rust
+use ranked_prolly_tree::{Tree, Result};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let mut tree = Tree::<64, _>::default();
+    let hello = String::from("hello").into_bytes();
```

Comment:
As we discussed I would recommend making higher level abstraction for keys so that layers below could take advantage of that in various ways. I think chucked nature is probably only thing that would matter.

Reply:
Filed: https://github.com/commontoolsinc/system/issues/272